src/olympia/files/tests/test_models.py

```diff
@@ -197,12 +197,12 @@ def test_generate_filename(self):
         file_ = File.objects.get(id=67442)
         assert (
             file_._meta.get_field('file').upload_to(file_, None)
-            == '15/3615/3615/a3615-2.1.072.zip'  # zip extension.
+            == '15/3615/3615/a3615-2.1.072.xpi'  # xpi extension.
         )
-        file_.is_signed = True
+        file_.is_signed = False
```

Comment:
I think I'd have just added a `file_.is_signed = False` above the assert 🤷

Reply:
Doesn't matter in the end, but that would have been smart :sweat_smile: